packages/financial-templates-lib/src/price-feed/CoinMarketCapPriceFeed.js

```diff
@@ -0,0 +1,168 @@
+const { PriceFeedInterface } = require("./PriceFeedInterface");
+const { parseFixed } = require("@uma/common");
+
+class CoinMarketCapPriceFeed extends PriceFeedInterface {
+  /**
+   * @notice Constructs the CoinMarketCapPriceFeed.
+   * @param {Object} logger Winston module used to send logs.
+   * @param {Object} web3 Provider from truffle instance to connect to Ethereum network.
+   * @param {String} apiKey CoinMarketCap API key.
+   * @param {String} symbol Cryptocurrency symbol. Example: BTC, ETH, DAI
+   * @param {String} convert Currency to use for calculating market quote. Example: PHP, USD
+   * @param {Integer} lookback How far in the past the historical prices will be available using getHistoricalPrice.
+   * @param {Object} networker Used to send the API requests.
+   * @param {Function} getTime Returns the current time.
+   * @param {Integer} minTimeBetweenUpdates Min number of seconds between updates. If update() is called again before
+   *      this number of seconds has passed, it will be a no-op.
+   * @param {Bool} invertPrice Indicates if prices should be inverted before returned.
+   * @param {Number} priceFeedDecimals Number of priceFeedDecimals to use to convert price to wei.
+   */
+  constructor(
+    logger,
+    web3,
+    apiKey,
+    symbol,
+    convert,
```

Comment:
Can we rename `convert` to `currency` since it looks like it has the same function as the `currency` value in `CoinGecko`? Even more accurate would be to rename both `currency` and `convert` to `quoteCurrency` to indicate that the price should be quoted in this currency

Reply:
@nicholaspai renamed both `CMC.convert` & `CG.currency` to `quoteCurrency`